### DISCUSSION TITLE
new standard `Shortestfile`

### DISCUSSION THREAD

**Discussion by amk-dev** [antiwork/shortest#208] [Ideas]
ok this is completly random, and might not be a good idea for complex tests. but just dropping this here, incase anyone thinks it's a good idea

but what if there's a `Shortestfile` (like Dockerfile) you put in your repo, where you list the tests you want to run in natural language.   shortest cli automatically picks it up and runs the tests.  

this will be 

```
for https://google.com,

1. visit google, search for "@antiwork/shortest" and makes sure the npm link and github link comes up in the results
2. make sure the signin button is visible in the top right corner

for https://youtube.com

1. example test 1
2. example test 2

```

`pnpm shortest` picks up the Shortestfile and runs it ?

**Comment by slavingia** [comment]
Seems quite outside the norm for standard dev workflow and maybe too probabilistic imo

**Comment by nazargladish** [comment]
A `.yaml` file could be more structured and natural for engineers, although makes tool more technical, and perhaps with less flexibility

**Reply by slavingia** [reply]
If this is wanted, just putting the tests directly in the existing config imo would be simplest.
